services/src/unit-test/java/uk/gov/moj/sdt/services/BulkSubmissionServiceTest.java

```diff
@@ -201,7 +290,7 @@ void testSaveBulkSubmission() throws IOException {
      */
     @Test
     void testSubmissionWithMultipleRequests() throws IOException {
-        final String rawXml = Utilities.getRawXml(PATH_TEST_RESOURCES, "testXMLValid3.xml");
+        final String rawXml = Utilities.getRawXml(PATH_TEST_RESOURCES, TEST_XML_VALID_2);
```

Comment:
The file that was originally loaded was testXMLValid3.xml, not testXMLValid2.xml.

Reply:
Done